code/modules/uplink/uplink_items/stealthy.dm

```diff
@@ -67,13 +67,12 @@
 	name = "Boxed Origami Kit"
 	desc = "This box contains a guide on how to craft masterful works of origami, allowing you to transform normal pieces of paper into \
 			perfectly aerodynamic (and potentially lethal) paper airplanes."
-	progression_minimum = 15 MINUTES
 	item = /obj/item/storage/box/syndie_kit/origami_bundle
-	cost = 14
+	progression_minimum = 10 MINUTES
```

Comment:
since no maintainer has shown interest yet, I guess I'll give my opinion.
If you want to bring the cost down, why do you also want to make it available quicker? There's not much explanation why this is needed too.

Reply:
I made it available at 10 minutes because the box of throwing weapons is 10 minutes, since I am comparing them as roughly equal in power, it makes sense to me that they are available at the same time.

Reply:
![image](https://user-images.githubusercontent.com/66163761/210213029-e6f41619-9c36-4f44-aee5-fcec6e2fbfb8.png)